respect-app-compose/src/commonMain/kotlin/world/respect/app/view/clazz/detail/ClazzDetailScreen.kt

```diff
@@ -0,0 +1,287 @@
+    onClickDismissInvite: (OneRosterUser) -> Unit
+) {
+    LazyColumn(
+        modifier = Modifier
+            .fillMaxSize()
+    ) {
+        item {
+            ListItem(
+                headlineContent = {
+                    Text(
+                        text = stringResource(
+                            resource = Res.string.description
+                        )
+                    )
+                },
+
+                /**Description field needed**/
+
+                /* supportingContent = {
+                     Text(
+                        uiState.clazzDetail?.description
+                     )
+                 }*/
+            )
+        }
+
+        item {
+            RespectFilterChipsHeader(
+                options = uiState.chipOptions.map { it.option },
+                selectedOption = uiState.selectedChip,
+                onOptionSelected = { onSelectChip(it) },
+                optionLabel = { it }
+            )
+
+            RespectListSortHeader(
+                activeSortOrderOption = uiState.activeSortOrderOption,
+                sortOptions = uiState.sortOptions,
+                enabled = uiState.fieldsEnabled,
+                onClickSortOption = onSortOrderChanged,
+            )
+        }
+
+        item {
+            ListItem(
+                headlineContent = {
+                    Text(
+                        text = stringResource(
+                            resource = Res.string.pending_invite
+                        )
+                    )
+                },
+                trailingContent = {
+                    Icon(
+                        imageVector = Icons.Outlined.KeyboardArrowDown,
+                        contentDescription = null,
+                        modifier = Modifier.size(24.dp),
+                    )
+                }
+            )
+        }
+
+        itemsIndexed(
+            uiState.listOfPending,
+            key = { index, pendingUser ->
+                pendingUser.sourcedId
+            }) { index, user ->
+            val roleName = user.roles.firstOrNull()?.role?.name
+                ?.lowercase()
+                ?.replaceFirstChar { it.uppercase() } ?: ""
+
+            ListItem(
+                modifier = Modifier
+                    .fillMaxWidth(),
+                leadingContent = {
+                    RespectPersonAvatar(
+                        name = user.givenName
+                    )
+                },
+                headlineContent = {
+                    Text(text = user.givenName + "(" + roleName + ")")
+                },
+                trailingContent = {
+                    Row {
+                        Icon(
+                            modifier = Modifier.size(24.dp).clickable {
+                                onClickAcceptInvite(user)
+                            },
+                            imageVector = Icons.Outlined.CheckCircle,
+                            contentDescription = null,
```

Comment:
These buttons are part of the UI - they need a content description for accessibility.